src/config/Tokens.js

```diff
@@ -432,3 +432,9 @@ export function getTokenBySymbol(chainId, symbol) {
 export function getWhitelistedTokens(chainId) {
   return TOKENS[chainId].filter((token) => token.symbol !== "USDG");
 }
+
+export function getVisibleTokens(chainId) {
+  return getWhitelistedTokens(chainId)
+    .filter((t) => !t.isWrapped)
```

Comment:
let's check both conditions inside single callback? in this specific case array is small and it won't affect performance much of course, but just a good thing to avoid double iteration over array imo

Reply:
Done!